src/main/java/ch/njol/skript/classes/data/SkriptClasses.java

```diff
@@ -700,6 +711,156 @@ public String toVariableNameString(VisualEffect e) {
+			.name("Config")
+			.description("A configuration (or code) loaded by Skript, such as the config.sk or aliases.",
+				"Configs can be reloaded or navigated to find options.")
+			.usage("")
+			.examples("the skript config")
+			.since("INSERT VERSION")
+			.parser(new Parser<Config>() {
+
+				@Override
+				public boolean canParse(ParseContext context) {
+					return false;
+				}
+
+				@Override
+				public String toString(Config config, int flags) {
+					@Nullable File file = config.getFile();
+					if (file == null)
+						return config.getFileName();
+					return Skript.getInstance().getDataFolder().getAbsoluteFile().toPath()
+						.relativize(file.toPath().toAbsolutePath()).toString();
+				}
+
+				@Override
+				public String toVariableNameString(Config config) {
+					return this.toString(config, 0);
+				}
+			}));
+
+		Classes.registerClass(new ClassInfo<>(Node.class, "node")
+			.user("nodes?")
+			.name("Node")
+			.description("A node (entry) from a script config file.",
+				"This may have navigable children.")
+			.usage("")
+			.examples("the current script")
+			.since("INSERT VERSION")
+			.parser(new Parser<Node>() {
+
+				@Override
+				public boolean canParse(ParseContext context) {
+					return false;
+				}
+
+				@Override
+				public String toString(Node node, int flags) {
+					return node.getPath();
+				}
+
+				@Override
+				public String toVariableNameString(Node node) {
+					return this.toString(node, 0);
+				}
+
+			}));
+
+		Classes.registerClass(new ClassInfo<>(Script.class, "script")
+				.user("scripts?")
+				.name("Script")
+				.description("A script loaded by Skript.",
+					"Disabled scripts will report as being empty since their content has not been loaded.")
+				.usage("")
+				.examples("the current script")
+				.since("INSERT VERSION")
+				.parser(new Parser<Script>() {
+					final Path path = Skript.getInstance().getScriptsFolder().getAbsoluteFile().toPath();
+
+					@Override
+					public boolean canParse(final ParseContext context) {
+						return switch (context) {
+							case PARSE, COMMAND -> true;
+							default -> false;
+						};
+					}
+
+					@Override
+					@Nullable
+					public Script parse(final String name, final ParseContext context) {
+						return switch (context) {
+							case PARSE, COMMAND -> {
+								@Nullable File file = ScriptLoader.getScriptFromName(name);
+								if (file == null || !file.isFile())
+									yield null;
+								yield ScriptLoader.getScript(file);
+							}
+							default -> null;
+						};
+					}
+
+					@Override
+					public String toString(final Script script, final int flags) {
+						return this.toVariableNameString(script);
+					}
+
+					@Override
+					public String toVariableNameString(final Script script) {
+						@Nullable File file = script.getConfig().getFile();
+						if (file == null)
+							return script.getConfig().getFileName();
+						return path.relativize(file.toPath().toAbsolutePath()).toString();
+					}
```

Comment:
```suggestion
					@Override
					public String toString(final Script script, final int flags) {
						@Nullable File file = script.getConfig().getFile();
						if (file == null)
							return script.getConfig().getFileName();
						return path.relativize(file.toPath().toAbsolutePath()).toString();
					}

					@Override
					public String toVariableNameString(final Script script) {
						return this.toString(script, 0);
					}
```
swapping toVariableNameString to call toString